reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/utils/PluginConfig.kt

```diff
@@ -67,7 +65,7 @@ class PluginConfig(
      *
      * @return [StringBuilder]
```

Comment:
Not a `StringBuilder` actually.

Reply:
it is diktat with auto generation, sorry((